cdc/owner/ddl_manager.go

```diff
@@ -208,7 +208,7 @@ func (m *ddlManager) tick(
 			return nil, nil, err
 		}
 		if !ok {
-			return nil, nil, nil
+			return nil, m.barrier(), nil
```

Comment:
Why return an empty barrier here?

Reply:
return nil cause the panic, and the barrier is initialized as empty I think.